### PR TITLE
Add ProxyFromEnvironment in downloadFile method

### PR DESCRIPTION
**Pull request by Lichmaker**
# Description

Add ProxyFromEnvironment in downloadFile method.

## Issue reference

Please reference the issue this PR will close: [#1043 ](https://github.com/dapr/cli/issues/1043)

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Review by hueifeng — APPROVED** [review]
lgtm

**Review by BjoernAtBosch — APPROVED** [review]
For me this fix solves the issue

**Comment by pravinpushkar** [comment]
@Lichmaker Could you please check the lint failures.

File: pkg/standalone/standalone.go [hunk 1]
```diff
@@ -1055,6 +1055,7 @@ func downloadFile(dir string, url string) (string, error) {
 			}).Dial,
 			TLSHandshakeTimeout:   15 * time.Second,
 			ResponseHeaderTimeout: 15 * time.Second,
+			Proxy: http.ProxyFromEnvironment,
```
Comment (pravinpushkar) [inline pkg/standalone/standalone.go:1058]: Here it is not correctly indented thats why the lint failure. Run `gofmt -w <filepath>` to correct this.

Comment (Lichmaker) [reply pkg/standalone/standalone.go]: Thank you for your correct.  :>

**Review by pravinpushkar — APPROVED** [review]
lgtm

@Lichmaker We might need to release a new CLI version with this fix. So, could you also please create a separate PR against branch `release-1.8`. This will be in addition to this PR.

**Comment by Lichmaker** [comment]
@pravinpushkar  Glad to do it. Please check it. 😃   https://github.com/dapr/cli/pull/1049

**Comment by mukundansundar** [comment]
@Lichmaker Can you please create a docs PR on how to use this ? 
Potentially add it in this page? https://docs.dapr.io/operations/troubleshooting/common_issues/
cc @greenie-msft WDYT?

**Review by mukundansundar — APPROVED** [review]
LGTM. Thanks for the contribution!

**Comment by Lichmaker** [comment]
> @Lichmaker Can you please create a docs PR on how to use this ? Potentially add it in this page? https://docs.dapr.io/operations/troubleshooting/common_issues/ cc @greenie-msft WDYT?

OK 😃  I'll try

**Comment by mukundansundar** [comment]
@Lichmaker Thanks for the change to include docs PR !!

**Pull request merged**